### PR TITLE
Docblock

### PR DESCRIPTION
**Pull request by palicao**

### CODE REVIEW

File: src/LessQL/Database.php [hunk 1]
```diff
@@ -37,6 +43,10 @@ function __call( $name, $args ) {
 	/**
 	 * Returns a result for table $name.
 	 * If $id is given, return the row with that id.
+	 *
+	 * @param $name
+	 * @param int|null $id
+	 * @return Result
```
Comment (morris) [inline src/LessQL/Database.php:49]: this may return a Result, a Row, or null

File: src/LessQL/Result.php [hunk 2]
```diff
@@ -375,6 +427,8 @@ function delete() {
 
 	/**
 	 * Return a new root result which selects all rows in this result by primary key
+	 *
+	 * @return $this
```
Comment (morris) [inline src/LessQL/Result.php:431]: This returns a new Result instance, not $this

**Comment by morris** [comment]
Looking good - however there seem to be some errors in the types. Could you go through this again?

**Comment by morris** [comment]
Merged it anyway, thank you so much for your contribution!

**Comment by palicao** [comment]
Sorry but I had no time to go through it and fix the errors... Thanks for accepting my contribution!

**Pull request merged**